homeassistant/components/poolsense/binary_sensor.py

```diff
@@ -17,12 +17,12 @@
 BINARY_SENSOR_TYPES: tuple[BinarySensorEntityDescription, ...] = (
     BinarySensorEntityDescription(
         key="pH Status",
-        name="pH Status",
```

Comment:
Incidentally, I'm adding support for pH sensors and numbers in https://github.com/home-assistant/core/pull/95928
You might be interested in that too.

Reply:
Yes and no, your proposal is afaik only for sensor and numbers, whilst this sensor is just an On/Off switch if the pH status is good.

Reply:
Gotcha. My Poolstation integration does show the exact value, which I why I'm adding it as a device class.

Reply:
Then that would be a sensor in sensor.py, not in this file